packages/web/src/components/buy-audio-modal/BuyAudioModal.tsx

```diff
@@ -1,11 +1,12 @@
-import React, { useCallback } from 'react'
+import { useCallback } from 'react'
 
 import { Modal, ModalContent, ModalHeader, ModalTitle } from '@audius/stems'
```

Comment:
I think we might have the icon for solana already, we are using it in the collectibles list items i believe.

Reply:
I think that one is slightly different!